src/nvme/tree.c

```diff
@@ -1541,6 +1546,17 @@ static int nvme_ns_init(struct nvme_ns *n)
 	return 0;
 }
 
+static void nvme_ns_set_chrdev_name(struct nvme_ns *n, const char *name)
+{
+	char chrdev_name[PATH_MAX];
+	int instance, head_instance;
+
+	sscanf(name, "nvme%dn%d", &instance, &head_instance);
```

Comment:
Please add error checking here. sscanf might fail.

Reply:
Sure, Thanks for the review.